src/data/main.py

```diff
@@ -18,51 +20,70 @@
                  "Jadot",
                  "Roussel",
                  "Arthaud",
-                 "Poutou"]
+                 "Poutou",
+                 "all"]
+
 
 # ########################################################################### #
 #                                 Functions                                   #
 # ########################################################################### #
 
 
 def add_data_args(parser):
-    # Specify to the data mentioning a specific candidats or aclImdb
-    parser.add_argument('--download',
-                        default='aclImdb',
+    # Specify which task to perform: download, make_dataset or load
+    parser.add_argument('--task',
+                        required=True,
+                        choices=['download', 'make-dataset', 'load-dataset'],
+                        help="Specify which task to perform.")
+
+    # Specify on which data to perform the task
+    parser.add_argument('--data',
+                        required=True,
+                        default=None,
```

Comment:
put twitter as default value since it is by far the most used command,
to avoid long CLI commands

Reply:
Je prefere pas. Pour moi, vaut mieux bien faire comprendre à l'user qu'il doit préciser le type de data quoi qu il arrive.

Reply:
With limited requests from Twitter API I also prefer not to put it as a default choice.